lib/autobuild/import/git.rb

```diff
@@ -706,18 +755,24 @@ def has_branch?(package, branch_name)
         end
 
         def has_local_branch?(package)
-            has_branch?(package, local_branch)
+            if !local_branch.nil?
+                has_branch?(package, local_branch)
+            else
+                false
+            end
         end
 
         def detached_head?(package)
             current_branch(package).nil?
         end
 
         private def remote_branch_to_ref(branch)
-            if branch.start_with?("refs/")
-                branch
-            else
-                "refs/heads/#{branch}"
+            if !branch.nil?
```

Comment:
Do a guard and then the rest

~~~
return unless branch

if branch.start_with?(...)
...
end
~~~